src/main/java/de/dagere/peass/ci/logs/RTSLogFileVersionReader.java

```diff
@@ -0,0 +1,107 @@
+package de.dagere.peass.ci.logs;
+
+import de.dagere.peass.ci.helper.VisualizationFolderManager;
+import de.dagere.peass.ci.logs.rts.RTSLogData;
+import de.dagere.peass.dependency.analysis.data.TestCase;
+import de.dagere.peass.dependency.traces.TraceWriter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class RTSLogFileVersionReader {
+
+   private static final Logger LOG = LogManager.getLogger(RTSLogFileVersionReader.class);
+   private final VisualizationFolderManager visualizationFolders;
+
+   private final String version;
+   private final Map<TestCase, RTSLogData> files = new LinkedHashMap<>();
+   private File testClazzFolder;
+   private File methodFile;
+   private TestCase test;
+   private boolean isParameterizedWithoutIndex = false;
+
+   public RTSLogFileVersionReader(VisualizationFolderManager visualizationFolders, String version) {
+      this.visualizationFolders = visualizationFolders;
+      this.version = version;
+   }
+
+   public Map<TestCase, RTSLogData> getClazzLogs(Map<File, String> testClazzFolders) {
+      testClazzFolders.forEach(this::getClazzLog);
+      return files;
+   }
+
+   private void getClazzLog(File testClazzFolder, String module) {
+      LOG.debug("Looking for method files in {}", testClazzFolder.getAbsolutePath());
+      File[] methodFiles;
+      methodFiles = testClazzFolder.listFiles();
+      if (methodFiles != null) {
+         for (File methodFile : methodFiles) {
+            LOG.debug("Looking for method log file in {}", methodFile.getAbsolutePath());
+            if (!methodFile.isDirectory()) {
+               this.testClazzFolder = testClazzFolder;
+               this.methodFile = methodFile;
+               String clazz = testClazzFolder.getName().substring("log_".length());
+               String method = methodFile.getName().substring(0, methodFile.getName().length() - ".txt".length());
+               test = new TestCase(clazz, method, module);
+
+               long count = Arrays.stream(methodFiles).map(File::getName).filter(methode -> methode.contains("(")).map(methode -> methode.substring(0, methode.indexOf("(")))
```

Comment:
For such statements, it would be nicer if each method call (`.map`, `.filter` etc.) would be on one line.

Reply:
it is fixed